schemas/DatabaseField.schema

```diff
@@ -23,41 +23,36 @@
     const fields = Object.keys(data);
 
     // Check allowed fields
-    const allow = allowed => {
-      for (const field of fields) {
-        if (!allowed.includes(field)) return false;
-      }
-      return true;
-    };
+    const allow = allowed => fields.every(f => allowed.includes(f));
```

Comment:
What do you think of using Set for `allowed` here?
Then this should be cleaner and maybe a bit faster.

Also, I'd suggest to rename this method to `checkFields`, `chechFieldConstraints` or something. Because `allow` suggest that this method somehow 'allows' fields provided.

Reply:
Using array is fine with me too. Though what about the renaming bit?

Reply:
mb `allowFields` because it allows subset of record fields:
```js
const fits = allowFields(['category', 'comment']);
if (!fits) return false;
```